manifest_staging/charts/aad-pod-identity/templates/mic-podsecuritypolicy.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.rbac.pspEnabled }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ template "aad-pod-identity-psp.mic.fullname" . }}
+  labels:
+    {{- include "aad-pod-identity.labels" . | nindent 4 }}
+spec:
+  volumes:
+    - hostPath
+    - secret
+  runAsUser:
+    rule: RunAsAny
+  privileged: true
```

Comment:
The rest of the spec fields are also required, otherwise the psp will fail to apply

```yaml
spec:
  allowPrivilegeEscalation: false
  fsGroup:
    ranges:
    - max: 65535
      min: 1
    rule: RunAsAny
  requiredDropCapabilities:
  - ALL
  seLinux:
    rule: RunAsAny
  supplementalGroups:
    ranges:
    - max: 65535
      min: 1
    rule: RunAsAny
  volumes:
    - hostPath
    - secret
  runAsUser:
    rule: RunAsAny
  privileged: false
```

- MIC doesn't require privilege mode

Reply:
done